package-lock.json

```diff
@@ -3840,61 +3840,46 @@
             }
         },
         "node_modules/@cypress/request": {
-            "version": "3.0.1",
-            "resolved": "https://registry.npmjs.org/@cypress/request/-/request-3.0.1.tgz",
-            "integrity": "sha512-TWivJlJi8ZDx2wGOw1dbLuHJKUYX7bWySw377nlnGOW3hP9/MUKIsEdXT/YngWxVdgNCHRBmFlBipE+5/2ZZlQ==",
+            "version": "3.0.6",
+            "resolved": "https://registry.npmjs.org/@cypress/request/-/request-3.0.6.tgz",
+            "integrity": "sha512-fi0eVdCOtKu5Ed6+E8mYxUF6ZTFJDZvHogCBelM0xVXmrDEkyM22gRArQzq1YcHPm1V47Vf/iAD+WgVdUlJCGg==",
             "dev": true,
+            "license": "Apache-2.0",
```

Comment:
Are all these changes needed?

Reply:
Not sure. I ran npm install to update package lock